edgedb/protocol/protocol.pyx

```diff
@@ -1104,6 +1104,17 @@ cdef class SansIOProtocol:
             in_dc.encode(buf, ())
             return
 
+        if self.protocol_version >= (0, 12):
+            if type(in_dc) is not ObjectCodec:
+                raise errors.QueryArgumentError(
+                    'unexpected query argument codec')
+
+            if args:
+                kwargs = {str(i): v for i, v in enumerate(args)}
+
+            in_dc.encode(buf, kwargs)
+            return
+
```

Comment:
Indent the below block under `else: ` for clarity.

Reply:
changed